HealthcareSystem/PatientWebApp/wwwroot/css/style_index.css

```diff
@@ -0,0 +1,17 @@
+﻿/* Style the button and place it in the middle of the container/image */
+.container .btn {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
+    background-color: #555;
+    color: white;
+    font-size: 16px;
+    padding: 12px 24px;
+    border: none;
+    cursor: pointer;
+    border-radius: 5px;
+}
```

Comment:
I kind of hate setting any colours and padding and whatever like this when you're using Bootstrap, and I feel like quite a bit is getting ignored anyway. I would just go for the text-primary btn-light classes, maybe one for padding if necessary but it might not be. Also, I'd ditch the border radius, you can barely see it but if I was a user and noticed it would just annoy me that it's inconsistent with every other button.